src/services/handleStorage.test.js

```diff
@@ -0,0 +1,45 @@
+import { getFromStorage, saveToStorage } from './handleStorage'
+
+describe('save to storage', () => {
+  it('should save dataset to storage', () => {
+    const mockSaveToStorage = jest.fn(saveToStorage)
```

Comment:
Kann man hier vielleicht Zeilen 5 bis 11 in ein before Each unters describe packen?